website/source/guides/load-balancing/nginx.html.md

```diff
@@ -0,0 +1,226 @@
+### Step 2: Deploy the Demo Web App
+
+We can now deploy our demo web application:
+
+```shell
+$ nomad run webapp.nomad 
+==> Monitoring evaluation "ea1e8528"
+    Evaluation triggered by job "demo-webapp"
+    Allocation "9b4bac9f" created: node "e4637e03", group "demo"
+    Allocation "c386de2d" created: node "983a64df", group "demo"
+    Allocation "082653f0" created: node "f5fdf017", group "demo"
+    Evaluation status changed: "pending" -> "complete"
+==> Evaluation "ea1e8528" finished with status "complete"
+```
+
+### Step 3: Create a Job for Nginx
+
+Create a job for Nginx and name it `nginx.nomad`. This will be our load balancer
+that will balance requests to the deployed instances of our web application.
+
+```hcl
+job "nginx" {
+  datacenters = ["dc1"]
+
+  group "nginx" {
+    count = 1
+
+    task "nginx" {
+      driver = "docker"
+
+      config {
+        image = "nginx"
+
+        port_map {
+          http = 80
+        }
+
+        volumes = [
+          "local:/etc/nginx/conf.d",
+        ]
+      }
+
+      template {
+        data = <<EOF
+upstream backend {
+{{ range service "demo-webapp" }}
+  server {{ .Address }}:{{ .Port }};
+{{ end }}
+}
+
+server {
+   listen 80;
+
+   location / {
+      proxy_pass http://backend;
+   }
+}
+EOF
+
+        destination = "local/load-balancer.conf"
+      }
+
+      resources {
+        network {
+          mbits = 10
+
+          port "http" {
+            static = 8080
+          }
+        }
+      }
+
+      service {
+        name = "nginx"
+        port = "http"
+      }
+    }
+  }
+}
+```
+- We are using Nomad's [template][template-stanza] to populate the load balancer
+  configuration for Nginx. The underlying tool being used is [Consul
+  Template][consul-template]. You can use Consul Template's documentation to
+  learn more about the [syntax][consul-temp-syntax] needed to interact with
+  Consul. In this case, we are going to query the address and port of our demo
+  service called `demo-webapp`.
+- We have statically set the port of our load balancer to `8080`. This will
+  allow us to query `nginx.service.consul:8080` from anywhere inside our cluster
+  so we can reach our web application.
+- Please note that although we have defined the template [inline][inline], we
+  can use the template stanza [in conjunction with the artifact
+  stanza][remote-template] to download an input template from a remote source
+  such as an S3 bucket.
+
+### Step 4: Run the Nginx Job
+
+We can now register our Nginx job:
+
+```shell
+$ nomad run nginx.nomad 
+==> Monitoring evaluation "45da5a89"
+    Evaluation triggered by job "nginx"
+    Allocation "c7f8af51" created: node "983a64df", group "nginx"
+    Evaluation status changed: "pending" -> "complete"
+==> Evaluation "45da5a89" finished with status "complete"
+```
+
+### Step 5: Make a Request to the Load Balancer
+
+If you query the Nginx load balancer, you should be able to see a response similar to the one shown below:
+
+```shell
```

Comment:
Can we use `console` instead of shell?

Reply:
Definitely open to this suggestion. Let's finalize on this as a team, and I'll use this going forward.